hivemind_listener/transformers.py

```
@@ -0,0 +1,204 @@
+        transformers are applied last.
+
+        A plugin of `priority` 1 will override any existing context keys
+        """
+        return sorted(self.loaded_plugins.values(),
+                      key=lambda k: k.priority, reverse=True)
+
+    def shutdown(self):
+        for module in self.plugins:
+            try:
+                module.shutdown()
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)